src/pages/dashboard.vue

```diff
@@ -74,13 +73,28 @@ const containRole = (roleType) => {
     party.roles.find(role => role.roleType === roleType && !role.cessationDate)
   )
 }
+
+// load information for the business or the bootstrap business,
+// and load the todo tasks, pending-review item, and filing history
 const loadBusinessInfo = async (force = false) => {
   const identifier = route.params.identifier as string
   if (identifier) {
     if (bootstrap.checkIsTempReg(identifier)) {
       // this is a business bootstrap (actual business does not exist yet)
       await bootstrap.loadBusinessBootstrap(identifier, force)
-      useBcrosTodos().loadBootstrapTask({ enabled: true, order: 0, task: bootstrapFiling.value } as TaskI)
+
+      // add the bootstrap item to the To Do, Pending or Filing History section.
+      if (bootstrap.isBootstrapTodo) {
+        useBcrosTodos().loadBootstrapTask({
+          enabled: true,
+          order: 0,
+          task: { filing: bootstrapFiling.value.filing }
+        } as TaskI)
+      } else if (bootstrap.isBootstrapFiling) {
+        useBcrosFilings().loadBootstrapFiling(bootstrapFiling.value)
+      } else if (bootstrap.isBootstrapPending) {
+        bootstrap.loadPendingFiling()
+      }
```

Comment:
This is the starting point for handling bootstrap business.
1) if the bootstrap filing is a todo item (e.g., draft filing, change requested), create todo items in Todo store. 
2) if the bootstrap filing is pending review (currently we only have pending continuation-in), create an item in BootstrapBusiness store and display it in the Pending section. 
3) if the bootstrap filing is rejected or future-effective, display it in the Recent Filing History section.